app/controllers/uploads_controller.rb

```diff
@@ -83,14 +88,23 @@ def upload_params
     @upload_params ||= params.require(:upload).permit(:csv_type, :skip_lines, :col_sep, :upload_file, :comment)
   end
 
+  def set_options
+    csv_type = @upload.csv_type
+    options = { skip_lines: @upload.skip_lines.try(:to_i), col_sep: @upload.col_sep }
+    unless defaults(csv_type)['force_simple_split'].nil? && defaults(csv_type)['strip_chars_from_headers'].nil?
+      options[:force_simple_split] = defaults(csv_type)['force_simple_split']
+      options[:strip_chars_from_headers] = defaults(csv_type)['strip_chars_from_headers']
+    end
```

Comment:
Same comment: Looks like `force_simple_split` and `strip_chars_from_headers` default to `nil` anyway, so I don't think you need these conditionals.